paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -462,6 +466,11 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
      */
     abstract fun setupLink(stripeIntent: StripeIntent)
 
+    suspend fun getLinkAccountStatus(): AccountStatus? =
+        linkConfiguration.value?.let {
+            linkLauncher.getAccountStatusFlow(it).first()
```

Comment:
Do we want to use `first` here? Doesn't the account status change after the user successfully log in? Does that impact the account status?

Reply:
This one is temporary until we submit the refactor in #5627 and make it all compose, then we'll collect the flow as state. Right now we need to check the current value at a few occasions, changes are not automatically handled because the UI is still not fully in Compose.
Removed this method and put the logic in `BaseAddPaymentMethodFragment` which is the only place that uses it. That part of the code will be cleaned up when moving to Compose.